### ISSUE TITLE
Should this be `np.random`?

### ISSUE THREAD

**Issue by wohlbier** [bkj/basenet#3]
Trying to profile this using Arm MAP lead me to find that the line referenced below might should be `_ = np.random.seed(seed + 789)`? The profiler was failing on this line.

https://github.com/bkj/basenet/blob/c61f558c7bd6341dcc1610135553c96bd7d2ca5b/basenet/helpers.py#L26

**Comment by bkj** [comment]
The line sets the seed for the python random package, which is used in
pytorch dataloaders.

**Comment by wohlbier** [comment]
Did something change with python random going from version 3.6 to 3.7?

**Comment by wohlbier** [comment]
Fat fingered environment.
